From 70e15ee4e1ef959986f929cb46599b09e424cb23 Mon Sep 17 00:00:00 2001
From: Alon Levy <alevy@redhat.com>
Date: Wed, 12 May 2010 16:44:40 -0300
Subject: [PATCH 15/15] spice-vmc: remove unused property "name"

RH-Author: Alon Levy <alevy@redhat.com>
Message-id: <1273682680-13503-1-git-send-email-alevy@redhat.com>
Patchwork-id: 9221
O-Subject: [RHEL-6 PATCH] spice-vmc: remove unused property "name"
Bugzilla: 576488
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

BZ 576488 (the general spicevmc bug)

This is a potential problem if a name is supplied from the command line.

Signed-off-by: Alon Levy <alevy@redhat.com>
---
 hw/spice-vmc.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/spice-vmc.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/hw/spice-vmc.c b/hw/spice-vmc.c
index a3661ae..72a292d 100644
--- a/hw/spice-vmc.c
+++ b/hw/spice-vmc.c
@@ -321,7 +321,6 @@ static VirtIOSerialPortInfo spice_vmc_info = {
     .qdev.props = (Property[]) {
         DEFINE_PROP_UINT32("nr", SpiceVMChannel, vserport.id,
                            VIRTIO_CONSOLE_BAD_ID),
-        DEFINE_PROP_STRING("name", SpiceVMChannel, vserport.name),
         DEFINE_PROP_END_OF_LIST(),
     },
 };
-- 
1.7.0.3

